### PR TITLE
Add Note

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: img/n/note.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...642 4.417187 0 8.143019 2.8936553 9.246464 6.6213642.415495 1.378742.551723 2.274073.551723 4.96347v22.764556c0 1.378742 0 2.757484-.275861 4.000053-.75947 3.448557-3.865465 6.553278-7.315435 7.312437-1.239674.275749-2.622386.275749-4.001693.275749zm73.14754 83.602808h-122.141884v-107.53504c0-1.38555.4836088-2.553225 1.4508265-3.520046l31.6729735-31.6599936c.967218-.9668213 2.138777-1.450232 3.518084-1.450232h85.5zm-140.6348137-125.6118746c-3.2456284 3.2442982-5.3844055 7.3158416-5.7999005 11.8673906-.0681139.755755-.1362278 1.858748-.1362278 2.965146v121.608417c0 1.586404.0681139 2.413649.1362278 3.036636.5551285 5.171132 5.244772 9.862258 10.4248357 10.413755.6198367.07149 1.4474209.136172 3.0344752.136172h143.8089223c1.59046 0 2.414639-.064682 3.037881-.136172 5.173253-.551497 9.866302-5.242623 10.418024-10.413755.068114-.622987.139634-1.450232.139634-3.036636v-165.82347c0-1.5864039-.07152-2.4170532-.139634-3.0332316-.551722-5.1779408-5.244771-9.8656624-10.418024-10.4171591-.623242-.068086-1.447421-.1395763-3.037881-.1395763h-99.575742c-1.103445 0-2.210296.0714903-2.966361.1395763-4.556821.4119204-8.626628 2.549821-11.872256 5.7907149z" fill="#41c9b4" fill-rule="evenodd"/></svg>
```
Comment (phallobst) [inline img/n/note.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="19.6" height="22.4" fill="none" stroke="#41c9b4" stroke-linecap="round" stroke-linejoin="round"><path stroke-width="2.8" d="M6.2 1.4h12V21H1.4V6.3Z"/><path stroke-width="2.2" d="M5.5 8.7h3.1v-3"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/n/note.com.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "note.mu"
+    ],
+    "contact": {
+      "facebook": "note.poc",
+      "twitter": "note_PR",
+      "email": "info@note.jp"
+    },
+    "keywords": [
+      "social"
+    ]
```
Comment (Carlgo11) [inline entries/n/note.com.json:15]: Looks like it's only in Japanese? A region tag might be good.
```suggestion
    ],
    "regions": [
      "jp"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/n/note.com.json (hunk 2 above)
Comment (ghost) [reply entries/n/note.com.json:15]: Note can be signed up from outside Japan. Language and region should not be confused.

Comment (Carlgo11) [reply entries/n/note.com.json:15]: I doubt it will be of much use on the Brazilian or German versions of 2fa.directory.

Comment (ghost) [reply entries/n/note.com.json:15]: I don't think so.

Comment (ghost) [reply entries/n/note.com.json:15]: Or maybe we should create a separate page for each language, separate from the region-specific pages we already have.

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
Unless there's activity within the next week I'll close the PR.

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**